hack/verify-style.sh

```diff
@@ -13,12 +13,7 @@ if [[ ! $(which go) ]]; then
 fi
 if [[ ! $(which golint) ]]; then
   echo "golint not found on PATH. To install:"
-  echo "go get -u github.com/golang/lint/golint"
```

Comment:
@squeed PTAL at this change, with GO Modules, we don't need dep so removed it.

But do we need a check for `which golint`? given that we are not doing any linting check in this script?

Reply:
oh, good point, installing golint is silly.